felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/Key.kt

```diff
@@ -29,6 +29,7 @@ enum class Key(val str: String) {
     NESTE_BEHOV("neste_behov"),
     IDENTITETSNUMMER("identitetsnummer"),
     INITIATE_ID("initiateId"),
+    INITIATE_EVENT("initiate-event"),
```

Comment:
flisespikk, men kanskje camelCase eller snake_style, ikke enda en variant ;)

Reply:
:) Det føles veldig rar for meg å ha snake case med lower case men jeg vil endre det for consistency :)